### ISSUE TITLE
Allow to config db protocol while installing package

### ISSUE THREAD

**Issue by daisy-ycguo** [apache/incubator-openwhisk-package-cloudant#177]
Although `changesWebAction.js` can accept db protocol from param, `installCatalog.sh` doesn't have a way to config it. It will cause the trigger `changes` cannot be created successfully when the database doesn't accept `https` connection.

So we need to make db protocol can be configured through `installCatalog.sh`.

**Comment by daisy-ycguo** [comment]
While, I also notice this package cannot manage those Cloudant db which doesn't support `https` connection. It has been hard coded in the action source files to use "https".
We should fix this too.

**Comment by jasonpet** [comment]
I do not follow your original comment about changesWebAction.  I agree with you about the hardcoded files for the other actions.  I use couchdb with http for changes and it works fine.

**Comment by daisy-ycguo** [comment]
@jasonpet, you are right. My original comment was wrong. changesWebAction works fine for couchdb with http. I will change the title and description to highlight the hardcoded files.
